CRM/Contact/Form/Task/PDFLetterCommon.php

```diff
@@ -418,7 +422,7 @@ public static function postProcess(&$form) {
     }
 
     $tee = NULL;
-    if (Civi::settings()->get('recordGeneratedLetters') === 'combined-attached') {
+    if ($isLiveMode && Civi::settings()->get('recordGeneratedLetters') === 'combined-attached') {
```

Comment:
Hmm.. this seems to be changing existing behaviour here? Maybe fixing a bug but we should be clear why we are making this change.

Reply:
Sharp eye @mattwire. :) 

I believe this does fix a bug.  Loosely, this fixes a crash when using "Preview" on CiviCase 5 with `combined-attached` policy. I *suspect* it also fixes a bug with CiviCase 4 (using "Preview"+`combined-attached` -- albeit with a different symptom), but I didn't carefully verify that the bug had existed before. However, I did retest several permutations after this fix.

Reply:
Fix looks sensible!